pkg/test/test_data/create-or-update/00-assert.yaml

```diff
@@ -7,12 +7,11 @@ status:
     kind: ProwJob
   # InitialNamesAccepted will change from False -> True once the CRD is ready.
   conditions:
-  - message: no conflicts found
-    reason: NoConflicts
+  - type: NonStructuralSchema
```

Comment:
this is a list and we need to assert the whole list. Since some version of kubernetes, this new condition appeared so it had to be added.

Reply:
:+1: